tests/integration/tests/test_clustering.py

```diff
@@ -90,6 +91,12 @@ def test_worker_nodes(instances: List[harness.Instance]):
     assert "worker" in util.get_local_node_status(joining_node)
     assert "worker" in util.get_local_node_status(other_joining_node)
 
+    # Expect all snap services to be up and running. Add a little sleep to
+    # ensure that services not just started but are also stable.
+    time.sleep(5)
+    for instance in instances:
+        util.check_snap_services(instance)
```

Comment:
Should we just move ``check_snap_services`` here? https://github.com/canonical/k8s-snap/blob/c51fe1347c9747f1ca6cdf771a9ac51d7b83303a/tests/integration/tests/test_util/util.py#L297